torchvision/prototype/transforms/functional/_meta.py

```diff
@@ -183,6 +183,10 @@ def clamp_bounding_box(
     return convert_format_bounding_box(xyxy_boxes, BoundingBoxFormat.XYXY, format)
 
 
+def _split_alpha(image: torch.Tensor) -> List[torch.Tensor]:
+    return torch.tensor_split(image, indices=(-1,), dim=-3)
```

Comment:
Frankly I reluctantly agreed with this to unblock the other PR. IMO there is very little value having a private method that does a one liner for an existing method on Core. I don't really mind this, I just think of this from the perspective of our practices. IMO the idiom of:
```
image, alpha = torch.tensor_split(image, indices=(-1,), dim=-3)
```
already sufficiently shows that we split the image and the alpha channel. The additional one liner, wrapper method doesn't add anything to the readability of the code. From that perspective, my preference is not to merge this, but I'm not going to insist if this is important to you.